core/payment-driver/gnt/src/gnt/faucet.rs

```diff
@@ -48,11 +48,12 @@ impl EthFaucetConfig {
     }
 
     pub async fn request_eth(&self, address: Address) -> GNTDriverResult<()> {
-        log::debug!("request eth");
+        log::info!("Requesting Eth from faucet");
         let client = awc::Client::new();
         let request_url = format!("{}/{}", &self.faucet_address, utils::addr_to_str(address));
 
         async fn try_request_eth(client: &awc::Client, url: &str) -> GNTDriverResult<()> {
+            log::debug!("url: {}", url);
```

Comment:
I believe we do not need to repeat this 6 times.. I'd place it right after `info!` log above (line 51)

Reply:
good one! updated